### PR TITLE
add 27. Remove Element

### PR DESCRIPTION
**Pull request by katsurada1**
Please check my code.

### CODE REVIEW

File: 27.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    def removeElement(self, nums: List[int], val: int) -> int:
+        """A function to remove all occurrences of val in nums in-place.
+        Args:
+            nums (List[int]): A given an integer array.
+            val (int): A given an integer.
+        Returns:
+            int: The elements after removing the elements equal to val.
+        """
+        
+        n=0
```
Comment (kobakobashu) [inline 27.py:15]: n = 0

File: 27.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        for i in nums:
```
Comment (kobakobashu) [inline 27.py:16]: for num in nums

**Review by kobakobashu — CHANGES_REQUESTED** [review]
👀

**Comment by katsurada1** [comment]
I modified the code.
Please check my code.

**Review by kobakobashu — APPROVED** [review]
LGTM

**Pull request merged**